src/pages/Wishlist/Wishlist.css

```diff
@@ -5,6 +5,8 @@ body {
 .wishlist-wrapper {
   width: 90%;
   margin: 5rem auto 7rem auto;
+  height: 100vh;
```

Comment:
as suggested above , if this is page wrapper then use min-height

Reply:
Done